app/controllers/SilhouetteController.scala

```diff
@@ -0,0 +1,97 @@
+package controllers
+
+import com.mohiva.play.silhouette.api.actions.{ SecuredActionBuilder, UnsecuredActionBuilder }
+import com.mohiva.play.silhouette.api.repositories.AuthInfoRepository
+import com.mohiva.play.silhouette.api.services.{ AuthenticatorService, AvatarService }
+import com.mohiva.play.silhouette.api.util.{ Clock, PasswordHasherRegistry }
+import com.mohiva.play.silhouette.api.{ EventBus, Silhouette }
+import com.mohiva.play.silhouette.impl.providers.{ CredentialsProvider, GoogleTotpProvider, SocialProviderRegistry }
+import javax.inject.Inject
+import models.services.{ AuthTokenService, UserService }
+import play.api.Logging
+import play.api.http.FileMimeTypes
+import play.api.i18n.{ I18nSupport, Langs, MessagesApi }
+import play.api.libs.mailer.MailerClient
+import play.api.mvc._
+import utils.auth.DefaultEnv
+
+import scala.concurrent.duration.FiniteDuration
+
+abstract class SilhouetteController(override protected val controllerComponents: SilhouetteControllerComponents)
+  extends MessagesAbstractController(controllerComponents) with SilhouetteComponents with I18nSupport with Logging {
+
+  def SecuredAction: SecuredActionBuilder[EnvType, AnyContent] = controllerComponents.silhouette.SecuredAction
+  def UnsecuredAction: UnsecuredActionBuilder[EnvType, AnyContent] = controllerComponents.silhouette.UnsecuredAction
+
+  def userService: UserService = controllerComponents.userService
```

Comment:
I'm not sure this opens the door for potential naming conflicts. Should we not namespace this? Maybe with an inner `Silhouette` object?

Reply:
I'd be worried about naming conflicts if this were in a library, but since it's in a seed project, it's user-modifiable code.  If people don't like it, they can change the names. :-)

Reply:
You're right :+1: